### ISSUE TITLE
Refactor compute_method_usage

### ISSUE THREAD

**Issue by cliffckerr** [fpsim/fpsim#142]
This line takes a lot of time, needs to be refactored using Numpy:
```py
        bool_list = ppl.alive * [sex == 0 for sex in ppl.sex] * [min_age <= age for age in ppl.age] * [age < max_age for age in ppl.age]
```

Also need to refactor e.g. the hard-coded minimum age

**Comment by MObrien-IDM** [comment]
@cliffckerr  is this still an issue with v2.0?

**Comment by cliffckerr** [comment]
@MObrien-IDM Yes, closed by #445
